frontend/public/testdata/searches.json

```diff
@@ -9,7 +9,7 @@
       "document_description": "The policy recognizes that climate change contributes significantly to Kenya's increasing vulnerability to disasters in the last two decades and affects seriously the lives and livelihoods of communities. The policy therefore aims to institutionalise mechanisms to address these disasters and associated vulnerabilities stressing the central role of climate change in any sustainable and integrated National Strategy for Disaster Management.\u00a0\u00a0The policy emphasises preparedness on the part of the government, communities and other stakeholders and proposes to establish and strengthen Disaster Management institutions, partnerships and networking. It proposes to mainstream Disaster Risk Reduction in the development process and strengthen the resilience of vulnerable groups.\u00a0\u00a0Disaster Risk Management encompasses a full continuum from preparedness, relief and rehabilitation, mitigation and prevention including t diversification of vulnerable livelihoods and coping mechanisms. Ministry of State for Special Programmes in the Office of President is appointed as the chief national co-ordinator.",
       "document_description_match": true,
       "document_geography_english_shortname": "Kenya",
-      "document_id": 1299,
```

Comment:
so, in the search results we will still have `document_id` (but it will now be an `import_id`), and the slug field will be `document_slug`

Reply:
Thanks - let me fix that